lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -89,6 +89,11 @@ e2function number entity:id()
 	return this:EntIndex()
 end
 
+e2function number entity:creationID()
+	if not IsValid(this) then return 0 end
```

Comment:
I disagree that the shorthand of `and or` (Lua's equivalent to the `? :` operator) is easier to read. It's definitely possible that it's faster than an if statement (I haven't checked), but no, I don't think it's easier to read. Does the coding style say anything about this?